cmd/api/src/model/filter.go

```diff
@@ -344,6 +352,6 @@ func (s QueryParameterFilterParser) ParseQueryParameterFilters(request *http.Req
 
 func NewQueryParameterFilterParser() QueryParameterFilterParser {
 	return QueryParameterFilterParser{
-		re: regexp.MustCompile(`([\w]+):([\w\d\--_]+)`),
+		re: regexp.MustCompile(`([~\w]+):([\w\d\--_]+)`),
```

Comment:
I created a test suite on regexr with both the existing predicates and the new one to verify it's working
regexr.com/8530l